src/pages/Homepage/iwd/IWDHomepage.vue

```diff
@@ -456,23 +440,23 @@ export default {
 				{
 					id: 90,
 					name: 'Eco-friendly loans',
-					url: 'https://www.kiva.org/lend/eco-friendly-loans',
+					url: '/lend-by-category/eco-friendly-loans',
 					image: {
 						url: 'https://www-kiva-org-0.freetls.fastly.net/img/w218h122/3e24bc6cf22a5be5f0dc766ff8251ef3.jpg'
 					},
 					retinaImage: {
 						url: 'https://www-kiva-org-0.freetls.fastly.net/img/w436h244/2116f8657fe23bc31cf54dfdb59413cb.jpg'
 					}
 				}
-			];
+			]
 			/* eslint-enable max-len */
-		}
+		};
 	},
 	computed: {
 		primaryCtaUrl() {
 			const date = getPdtDate();
 			const march8 = 1583654400000; // epoch time of march 8.
-			return date.valueOf() < march8 ? '/monthlygood?category=women' : '/women';
+			return date.valueOf() < march8 ? '/monthlygood?category=women' : '/lend-by-category/women';
```

Comment:
MG landing page doesn't currently accept an incoming category param. I'll make a ticket and fix that today.